xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -883,6 +926,7 @@ def main():
     parser.add_argument('--graph_limit', help='Graph limit parameters')
 
     args = parser.parse_args()
+    eprint(" ".join(sys.argv))
```

Comment:
This might be a leftover, it should be probably removed

Reply:
Right, removed.